test/integration/apiserver/cel/validatingadmissionpolicy_test.go

```diff
@@ -0,0 +1,942 @@
+		t.Fatal(err)
+	}
+	autofailBinding := withBindingExistsLabels([]string{"autofail-binding-label"}, policy, makeBinding("autofail-binding", "test-policy", "autofail-params"))
+	if _, err := client.AdmissionregistrationV1alpha1().ValidatingAdmissionPolicyBindings().Create(context.TODO(), autofailBinding, metav1.CreateOptions{}); err != nil {
+		t.Fatal(err)
+	}
+
+	autoPassParams := makeConfigParams("autopass-params", map[string]string{
+		"autofail":    "false",
+		"conditional": "false",
+	})
+	if _, err := client.CoreV1().ConfigMaps("default").Create(context.TODO(), autoPassParams, metav1.CreateOptions{}); err != nil {
+		t.Fatal(err)
+	}
+	autopassBinding := withBindingExistsLabels([]string{"autopass-binding-label"}, policy, makeBinding("autopass-binding", "test-policy", "autopass-params"))
+	if _, err := client.AdmissionregistrationV1alpha1().ValidatingAdmissionPolicyBindings().Create(context.TODO(), autopassBinding, metav1.CreateOptions{}); err != nil {
+		t.Fatal(err)
+	}
+
+	condpassParams := makeConfigParams("condpass-params", map[string]string{
+		"autofail":    "false",
+		"conditional": "true",
+		"check":       "prefix-",
+	})
+	if _, err := client.CoreV1().ConfigMaps("default").Create(context.TODO(), condpassParams, metav1.CreateOptions{}); err != nil {
+		t.Fatal(err)
+	}
+	condpassBinding := withBindingExistsLabels([]string{"condpass-binding-label"}, policy, makeBinding("condpass-binding", "test-policy", "condpass-params"))
+	if _, err := client.AdmissionregistrationV1alpha1().ValidatingAdmissionPolicyBindings().Create(context.TODO(), condpassBinding, metav1.CreateOptions{}); err != nil {
+		t.Fatal(err)
+	}
+
+	// allow time for reconciliation
+	time.Sleep(time.Second)
+
+	autofailingSecret := &v1.Secret{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "autofailing-secret",
+			Labels: map[string]string{
+				"paramIdent":             "someVal",
+				"autofail-binding-label": "true",
+			},
+		},
+	}
+	_, err = client.CoreV1().Secrets("default").Create(context.TODO(), autofailingSecret, metav1.CreateOptions{})
+	if err == nil {
+		t.Fatal("expected secret creation to fail due to autofail-binding")
+	}
+	checkForFailedRule(t, err)
+	checkFailureReason(t, err, metav1.StatusReasonInvalid)
+
+	autopassingSecret := &v1.Secret{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "autopassing-secret",
+			Labels: map[string]string{
+				"paramIdent":             "someVal",
+				"autopass-binding-label": "true",
+			},
+		},
+	}
+	if _, err := client.CoreV1().Secrets("default").Create(context.TODO(), autopassingSecret, metav1.CreateOptions{}); err != nil {
+		t.Fatalf("expected secret creation to succeed, got: %s", err)
+	}
+
+	condpassingSecret := &v1.Secret{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "prefix-condpassing-secret",
+			Labels: map[string]string{
+				"paramIdent":             "someVal",
+				"condpass-binding-label": "true",
```

Comment:
I think I'm just missing something, but where is the expression checking for the `condpass-binding-label`? 
```
Expression: "params.data.autofail != 'true' && (params.data.conditional == 'false' || object.metadata.name.startsWith(params.data.check))",
```

Reply:
Oh, this is passing because of the name prefix, not the label